src/components/v5/frame/PageLayout/types.ts

```diff
@@ -4,7 +4,8 @@ import { type PageHeaderProps } from './partials/PageHeader/types.ts';
 
 export interface PageLayoutProps {
   sidebar: ReactNode;
-  headerProps: PageHeaderProps;
+  headerProps?: PageHeaderProps;
   topContent?: ReactNode;
   header?: ReactNode;
+  mobileAndDesktopLayout?: boolean;
```

Comment:
Can we rename this property into something like `isSmPaddingEnabled` or `applySmPadding` or `isSmStylingEnabled` if we want to be a bit more general? I feel `mobileAndDesktopLayout` doesn't convey enough information as in why to use it

Reply:
Thanks for reviewing @mmioana !

> When you complete the create profile/create colony flow, the bubble next to the Complete step is invisible

I shall investigate and fix this, great spot! 👌 

> Can we rename this property into something like isSmPaddingEnabled or applySmPadding or isSmStylingEnabled if we want to be a bit more general? I feel mobileAndDesktopLayout doesn't convey enough information as in why to use it

Tbh none of those variable names also convey what I'm trying to achieve here hahaha 😂  variable names are so difficult! It's not really a matter of applying padding on `sm`. It's a matter of varying the breakpoints primarily between mobile and desktop because by default, we are varying between `md` and everything above `md`. Happy to hear another batch of variable name suggestions and from everyone as well!

Reply:
> There is no header padding if you go to http://localhost:9091/go/{COLONY_NAME_HERE}

As for this, I'll see if this is relevant to my PR. If it isn't I will open a separate issue 👌 

@mmioana

Reply:
@mmioana I've given it a longer name now 😂 I dunno if you're okay with it but would like to get your thoughts tomorrow!